Modules/System Tests/MicrosoftGraph/src/DummyHttpResponseMessage.Codeunit.al

```diff
@@ -0,0 +1,30 @@
+codeunit 135144 "Dummy - HttpResponseMessage" implements IHttpResponseMessage

```

Comment:
Minor: A few files are missing the OSS header.

Reply:
Added the OSS Header